### DISCUSSION TITLE
Again: Relationships after uploading a csv to Budibase

### DISCUSSION THREAD

**Discussion by JohnnyFabritz** [Budibase/budibase#8504] [Ideas]
I imported two data tables in my Budibase DB, worked well. (These tables have been exported from an old filemaker DB). So, in each tables are matching fields, called "Auftrag-ID", corresponding to each other and formerly used to link these two tables.

I have read, that there is no way to "simple" relate the tables (and thats a pitty!), but could it be possible, to automate establishing the relations by a kind of action, automation, loop the whole table?

Has somebody a hint to do this?
 Thanks a lot!
 Johnny

**Screenshots**

**Comment by andz-bb** [comment]
hey @JohnnyFabritz, this was discussed recently in this thread: #8396

Unfortunately there's not currently any functionality to do this in Budibase. I think this is for performance reasons when importing large CSVs with many rows. It's possible that this could be added down the line if there are updates to Budibase DB to better support relationships but until then I don't think it's possible, sorry.
